raiden/network/proxies/token_network.py

```diff
@@ -43,6 +48,11 @@
 log = structlog.get_logger(__name__)  # pylint: disable=invalid-name
 
 
+def get_gas_limit(given, precalculated):
```

Comment:
A more general
```
none_safe_max(*args):
  return max(arg or 0 for arg in args)
```
would avoid misconceptions about the function doing anything related to gas (because it doesn't, right?).

Reply:
I pulled in the `GAS_FACTOR` as you proposed, so the name makes more sense now.